### ISSUE TITLE
8x10 photograph work

### ISSUE THREAD

**Issue by theresaanna** [khandelwal/AskShashank#38]
Question: should your 8x10 have text on it, a la the Sun King?

**Comment by jehlers** [comment]
Answer: Yes

**Comment by theresaanna** [comment]
![sassy sun king](http://i.snag.gy/qMp3n.jpg)

**Comment by ascott1** [comment]
Desktop wallpaper size plz. K, thnx.

**Comment by theresaanna** [comment]
Unfortunately it is cropped to a 4x5 ratio because @khandelwal offers 8x10s. He may consider distributing the high resolution image if you purchase his signed 8x10 package.

**Comment by ascott1** [comment]
I would like to buy the super deluxe ultra kickstarter backer package that contains the glossy signed 8X10, high resolution desktop wallpaper, t-shirt, and life-size cardboard cutout.

**Comment by stephanieosan** [comment]
:heart: :heart: :heart:

**Comment by rjmajma** [comment]
Stiiiiiiiiill waiting on this.

**Comment by michellehood** [comment]
I believe we are still waiting on this, right @rjmajma

**Comment by rjmajma** [comment]
This is correct. I believe others have also purchased said 8x10, but also have yet to receive theirs.
